foundations/the_front_end/html_css_basics.md

```diff
@@ -30,9 +29,10 @@ Look through these now and then use them to test yourself after doing the assign
 
 <div class="lesson-content__panel" markdown="1">
  
-  1. Dive right into learning HTML and CSS with [freeCodeCamp's interactive tutorials](https://www.freecodecamp.org/learn). Do the entire section labeled "Responsive Web Design Certification".  It's long and, if you're new, will take you at least a few days to get through, but it will give you a really solid foundation to build on moving forward. They have some projects at the end. They're good practice, but you can consider them optional. (A large part of their proposed '300 hours' for completion comes from these projects. Thus, it probably won't take you that long.)
+  1. Dive right into learning HTML and CSS with [freeCodeCamp's interactive tutorials](https://www.freecodecamp.org/learn). Do the entire section labeled "Responsive Web Design Certification" only. It's long and, if you're new, might take you at least a few days to get through, but it will give you a really solid foundation to build on moving forward. They also have some projects at the end of this section. They're good practice, but you can consider them optional. (A large part of their proposed '300 hours' for completion comes from these projects, so it probably won't take you that long if you only do the exercises).
+      * If you are using a browser add-on such as Dark Reader (or a similar add-on that creates a "dark mode" for a website), it may interfere with certain exercises. If your solution isn't accepted because of a color requirement not being met when your solution is actually correct, try disabling the browser add-on and submitting it again. An example where this issue may creep up is in the "Use CSS Selectors to Style Elements" exercise in the Basic CSS course.
```

Comment:
I would suggest shortening this to the below and doing **Note:** instead of the bullet point/indent. just make it the next line in '1.' paragraph. 

**Note:** If you are using a browser add-on such as Dark Reader (or a similar add-on that creates a "dark mode" for a website), it may interfere with certain exercises. If your solution isn't accepted , try disabling the browser add-on and submitting it again.